backend/routes.js

```diff
@@ -137,9 +163,23 @@ export default class Server {
             );
             res.send(204);
         });
-        server.listen(
-            parseInt(getSetting('PORT'), 10)
-        );
+
+        /*
+         * For some reason restartWithSSL() is triggered twice even if I
+         * properly delete the interval that is calling it ... TODO: investigate.
+         * For now a workaround is to catch the error when in use.
+         * There is nothing else besides the app that starts the server so
+         * there should be no real conflict or need to debug other in use
+         */
+        server.on('error', (e) => {
+            if (e.code == 'EADDRINUSE') {
```

Comment:
this can be a pretty common error when attempting to run these servers yourself or when services are managing restarting these servers automatically or when the machine boots. So we should figure out why this is happening so that we can properly quit with an unhandeled exception if this happens.